images/echoheaders/Makefile

```diff
@@ -1,11 +1,64 @@
-all: push
+# Copyright 2017 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-# TAG 0.0 shouldn't clobber any release builds
-TAG = 1.7
-PREFIX = gcr.io/google_containers/echoserver
+TAG = 1.8
+REGISTRY = gcr.io/google_containers
+ARCH ?= $(shell go env GOARCH)
+ALL_ARCH = amd64 arm ppc64le
```

Comment:
There's no workaround for this right now but I'm trying to find an alternative and remove the lua dependency.